elements/pfe-sass/functions/_custom-properties.scss

```
@@ -20,9 +20,12 @@
 ///         }
 ///         ```
 /// ===========================================================================
-@function pfe-get-from-map($category, $key, $map, $fallback: null) {
+@function pfe-get-from-map($category, $key, $map, $fallback: null, $repo-name: $repo) {
     // Start building the variable declaration
     $var-declaration: "--#{$repo}-theme--";
```

Because you're mapping $repo-name to $repo in the function, you can replace this with:

```suggestion
    $var-declaration: "--#{$repo-name}-theme--";
```

And delete the following 3 lines.